data/virt_autotest/guest_params_xml_files/alp_kvm_hvm_x86_64_imported_disk_vnet_template.xml

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0"?>
 <guest>
     <guest_os_name>alp</guest_os_name>
-    <guest_version>0.1</guest_version>
-    <guest_version_major>0</guest_version_major>
-    <guest_version_minor>1</guest_version_minor>
```

Comment:
Does removing ```guest_version_major``` and ```guest_version_minor``` help your test ? If not, you can keep them.

Reply:
They are not used at all for current alp GI.